crates/red_knot_python_semantic/src/types/call/bind.rs

```diff
@@ -170,11 +468,12 @@ impl<'a, 'db> IntoIterator for &'a mut Bindings<'db> {
 /// overloads, we store this error information for each overload.
 ///
 /// [overloads]: https://github.com/python/typing/pull/1839
-#[derive(Debug, Clone, PartialEq, Eq)]
-pub(crate) struct CallableBinding<'db> {
+#[derive(Debug, Clone)]
```

Comment:
Nit: Is it still necessary that all these types are `Clone`?

Reply:
Nope! Removed